tutorials/Tutorial9_DPR_training.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>Please check if the displayed error here is actually a bug. Seems like the tokenizer class cannot be inferred correctly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>This has been fixed</p>